CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
-## TBD
-* Updated `hotels-oss-parent` version to 4.2.0 (was 4.0.0). Contains updates to the copyright header.
+## [15.0.0] - TBD
+
+### Changed
```

Comment:
@massdosage adding a comment here to discuss
>>This might actually need one more change. Spring seems to be converting true in the config to a Boolean, which throws a ClassCastException when the replica table is created. This can be avoided by putting "true" in the config instead to force it in to a String, or I can add a check to cast properly to a String when the transformation is applied. What do people think?

>Where in the config is this boolean?

For example:
```
transform-options:
  table-properties:
    '[some.property]': true
```
Spring will read that property in to a Boolean, and then even though the map is cast to `Map<String, String>`, it remains a Boolean.  It's only when the table is created in Hive that the `ClassCastException` happens.

So we could validate before that happens, or just tell the user to wrap their values in quotes

Reply:
I kind of now recall us discussing this at some point (?) and I think we thought this is up to the user to enter their config correctly and not for Circus Train to decide.

Reply:
As much as I like to have proper types wherever possible I think we don't really want to have to consider all the types that could be passed into these maps so we could just document that the keys and values need to be **Strings** and then the end users can do conversion from Strings into whatever types they need themselves in their transforms.

Reply:
Okay cool, I've added to the readme. Thanks!